Sources/MapboxDirections/Intersection.swift

```diff
@@ -14,7 +14,13 @@ public struct Intersection {
                 outletIndexes: IndexSet,
                 approachLanes: [LaneIndication]?,
                 usableApproachLanes: IndexSet?,
-                outletRoadClasses: RoadClasses?) {
+                outletRoadClasses: RoadClasses?,
+                tollCollection: TollCollection?,
+                tunnelName: String?,
+                restStop: RestStop?,
+                isUrban: Bool?,
+                administrationRegionIndex: Int?,
```

Comment:
Adding arguments to an initializer is a backwards-incompatible change. To mitigate that change, give each argument a default value.